starfish/imagestack/imagestack.py

```diff
@@ -62,14 +62,17 @@ class ImageStack:
         retrieve a slice of the image tensor
     set_slice(indices, data, axes=None)
         set a slice of the image tensor
-    apply(func, split_by={Indices.X, Indices.Y}, in_place=False, verbose=False, n_processes=None)
-        split the image tensor into chunks and apply a function across each of the chunks to yield
-        an image tensor
-    transform(func, split_by={Indices.X, Indices.Y}, verbose=False, n_processes=None)
-        split the image tensor into chunks and apply a function across each of the chunks
+    apply(func, group_by={Indices.ROUND, Indices.CH, Indices.Z}, in_place=False, verbose=False,
+            n_processes=None)
+        split the image tensor along one or more axes and apply a function across each of the
+        components to yield an image tensor
+    transform(func, group_by={Indices.ROUND, Indices.CH, Indices.Z}, verbose=False,
+            n_processes=None)
+        split the image tensor along one or more axes and apply a function across each of the
+        components
```

Comment:
```suggestion
        components. Results are returned as a List with length equal to the number of times 
        the image tensor is split.
```

Reply:
The return style is the key difference between apply and transform -- since sphinx doesn't let us state return values, I think it's important to be explicit here.

Reply:
applied.